### DISCUSSION TITLE
Pre-built dashboards issue

### DISCUSSION THREAD

**Discussion by JumpyWizard404** [Security-Onion-Solutions/securityonion#9032] [Unsupported Versions]
Good afternoon anyone, we just deployed a test version of security onion, and have noticed a lot of the dashboards do not have any data in them mainly data that look into indexes such as the dashboard for Connections - Top Source IPs (namely dashboards that don't *:so-* as an index do not get any data and the system comes with dashboards that use logstash* as the source index, is this by design just to test the system, and if so is it by design to build custom dashboards from the ground up.

**Comment by reyesj2** [comment]
Have you ingested any data using ```so-import-pcap``` or by using a TAP/SPAN port?

**Reply by JumpyWizard404** [reply]
Hi @reyesj2 we are using Mirror Port (Suricata)

**Comment by dougburks** [comment]
Are you looking at the old 16.04 dashboards in Kibana? From https://docs.securityonion.net/en/2.3/kibana.html#dashboards:
> We’ve included the old 16.04 dashboards in case you performed an in-place upgrade and have any old 16.04 data. These dashboards are named with the z16.04 prefix and will only show old 16.04 data from the old *:logstash-* indices. The new Security Onion 2 dashboards are all named with the Security Onion prefix and they should be used for any new data stored in the new *:so-* indices.

Also, check out our dashboards built into SOC!
https://docs.securityonion.net/en/2.3/dashboards.html
